doc.go

```diff
@@ -75,6 +75,12 @@ The following options can be specified:
 	code blocks. Non-initializer code blocks in the grammar end up as methods on the
 	*current type, and this option sets the name of the receiver (default: c).
 
+	-alternate-entrypoints=RULE[,RULE...] : string, comma-separated list of rule names
+	that may be used as alternate entrypoints for the parser, in addition to the
+	default entrypoint (the first rule in the grammar) (default: none).
+	Such entrypoints can be specified in the call to Parse by passing an
+	Entrypoint option that specifies the alternate rule name to use.
```

Comment:
Maybe we should mention, that this flag is only useful/necessary, if the `-optimize-grammar` flag is used as well. Otherwise it does not have an impact.